pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -54,18 +53,20 @@ buildDotnetModule rec {
   '';
 
   makeWrapperArgs = [
-    "--prefix PATH : ${lib.makeBinPath [ desktop-file-utils ]}"
-    "--set APPIMAGE ${placeholder "out"}/bin/${meta.mainProgram}" # Make associating with nxm links work on Linux
+    # Make associating with nxm links work on Linux
+    "--set APPIMAGE ${placeholder "out"}/bin/${finalAttrs.meta.mainProgram}"
   ];
 
+  buildInputs = [ desktop-file-utils ];
```

Comment:
Can you explain the reasoning for this change? The package only contains binaries, putting this in `buildInputs` seems like a no-op to me.